vector/v.in.ogr/main.c

```diff
@@ -1053,13 +1053,20 @@ int main(int argc, char *argv[])
                 if (key_idx[layer] > -1 && key_idx[layer] == i)
                     continue; /* skip defined key (FID column) */
 
-		i_out++;
-                
 		Ogr_field = OGR_FD_GetFieldDefn(Ogr_featuredefn, i);
 		Ogr_ftype = OGR_Fld_GetType(Ogr_field);
 
 		G_debug(3, "Ogr_ftype: %i", Ogr_ftype);	/* look up below */
 
+                /* skip columns with unsupported data type */
+                if (Ogr_ftype == OFTBinary) {
+                    G_warning(_("Datatype of column <%s> is not supported. "
+                    "Omitting that column."), OGR_Fld_GetNameRef(Ogr_field));
+                    ncols_out = ncols_out - 1;
+                    continue;
+				}
```

Comment:
Please correct the indent.

Reply:
OK, I changed the warning message and tired to follow:
https://trac.osgeo.org/grass/wiki/Submitting/C#Indentation
regarding indent.
However, runing
```
bash utils/grass_indent.sh vector/v.in.ogr/main.c
```
gave me:
```
indent: vector/v.in.ogr/main.c:1464: Error:Stmt nesting error.
indent: vector/v.in.ogr/main.c:1786: Error:Stmt nesting error.
indent: vector/v.in.ogr/main.c:2049: Error:Stmt nesting error.
```
So, I kept only the changed indent for the block I added. I hope that is according to the guidelines. If other indent changes are needed I would appreciate a pointer to what should be done specifically...

Reply:
GNU indent can handle that file.
I was first thinking if the bad indent on line 1067, but keeping grass_indents changes for only added code is good in this case.

Reply:
While most code for re-indented and fixed to indent without errors, the v.in.ogr nesting error still remains (https://github.com/OSGeo/grass/issues/1630#issuecomment-1295442400).

Reply:
> GNU indent can handle that file.

I meant to say "GNU indent **can't** handle that file."